### ISSUE TITLE
Crashes occurring on LG devices running Android 8 and 9

### ISSUE THREAD

**Issue by pablobaxter** [pablobaxter/Harmony#38]
There is a crash that occurs when starting multiple Harmony preferences in LG devices, specifically in Android 8 and 9.

```
Caused by java.util.ConcurrentModificationException
       at java.util.ArrayList$Itr.next(ArrayList.java:860)
       at android.os.FileObserver.startWatching(FileObserver.java:196)
       at com.frybits.harmony.HarmonyImpl$isLoadedTask$1.call(HarmonyImpl.java:172)
       at com.frybits.harmony.HarmonyImpl$isLoadedTask$1.call(HarmonyImpl.java:72)
       at java.util.concurrent.FutureTask.run(FutureTask.java:266)
       at android.os.Handler.handleCallback(Handler.java:790)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:164)
       at android.os.HandlerThread.run(HandlerThread.java:65)
```

**Comment by pablobaxter** [comment]
This issue is reproducible on an LG device. Will work on a fix.

**Comment by pablobaxter** [comment]
Crash issue fixed in #39
